src/include/brain/catalog_sync_brain_job.h

```diff
@@ -0,0 +1,126 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// catalog_sync_brain_job.h
+//
+// Identification: src/include/brain/catalog_sync_brain_job.h
+//
+// Copyright (c) 2015-2018, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+#pragma once
+#include <vector>
+#include <string>
+#include <pqxx/pqxx>
+#include <algorithm>
+#include "concurrency/transaction_manager_factory.h"
+#include "brain/brain.h"
+#include "catalog/catalog.h"
+#include "catalog/table_catalog.h"
+#include "storage/data_table.h"
+#include "catalog/schema.h"
+#include "type/value_factory.h"
+#include "catalog/query_history_catalog.h"
+#include "catalog/query_metrics_catalog.h"
+
+namespace peloton {
+namespace brain {
+class CatalogSyncBrainJob : public BrainJob {
```

Comment:
Like I said in [this comment](https://github.com/cmu-db/peloton/pull/1283/files/412ebb9785bade124e610924fb2155275e373536#diff-e783af558990ff366767d25fe307af88):
1. Please add some brief comments to the functions in this class
2. I would move most of the function definitions in this class to a cpp file since they're more than the size of a function that you want to inline. And the reduce the includes.

Reply:
Will do